vcd/resource_vcd_network_isolated_v2_nsxt_test.go

```diff
@@ -157,3 +159,262 @@ resource "vcd_network_isolated_v2" "net1" {
   prefix_length = 24
 }
 `
+
+func TestAccVcdNetworkIsolatedV2NsxtMigration(t *testing.T) {
```

Comment:
Please add a small comment above function on what kind of migration this test is testing in particular to help fresh reader, as the word "migration" is quite overused these days (i.e. provides no context).

Reply:
Added a more detailed comment here and also used the opportunity to add a few more steps (additionally verifying `import`)